crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -34,6 +34,7 @@ def upgrade
       case @version
       when "1.9"
         knife_files19
+        reset_target_platform_for_nodes
```

Comment:
Is this really covered by the commit message?  ICBW but it looks to me like you are mixing an "extract method" refactoring (which I am almost always in favour of) with some changes in behaviour.